### ISSUE TITLE
formattedNumber useGrouping not working

### ISSUE THREAD

**Issue by EnriqueDev** [joshswan/react-native-globalize#19]
Trying to use useGrouping on FormattedNumber has no effect at all.

**Comment by joshswan** [comment]
Just published v1.0.0 with a fix in the CLDR data. The "en-US" data being included previously was incorrect. That has been removed in favor of "en", which is the appropriate locale for the US anyway. Other locales, such as "en-GB" should have worked correctly.

**Comment by EnriqueDev** [comment]
thank you :)

**Comment by cmmouritsen** [comment]
I'm not positive this is related, but I'm now seeing USD currency formatted with `US$` in front instead of just `$`.

**Comment by joshswan** [comment]
@cmmouritsen What locale are you using? As far as I can see `en` doesn't have the "US" in front, while `en-GB` and other do. I think that makes sense, no?

**Comment by cmmouritsen** [comment]
```
<FormattedWrapper locale="en-US" localeFallback={true}>
</FormattedWrapper>
```

**Comment by joshswan** [comment]
Looks like the `localeFallback` was causing `en-US` to fall back to `en-GB` now that `en-US` is gone in favor of just `en`. I just published v1.0.1 that should fix that hopefully.

**Comment by cmmouritsen** [comment]
That fixed it. Thanks!
